Add pygame.mouse.get_just_pressed/released() (#2836)
pygame-community/pygame-ce: docs/reST/ref/mouse.rst

   .. ## pygame.mouse.get_pressed ##

.. function:: get_just_pressed

   | :sl:`get the most recently pressed buttons`
   | :sg:`get_just_pressed() -> (left_button, middle_button, right_button, x1_button, x2_button)`

>    Very similar to :func:`pygame.mouse.get_pressed()`, returing a tuple

Reviewer: Typo: "returing"
```suggestion
   Very similar to :func:`pygame.mouse.get_pressed()`, returning a tuple
```

   of length 5 with the important difference that the buttons are 
   True only in the frame they start being pressed. This can be convenient

>    for checking the buttons pressed "this frame" but for more precise results
>    and to have the correct order the pygame.MOUSEBUTTONDOWN event could be
>    preferred.

Reviewer: I think this is more clear
```suggestion
   for checking the buttons pressed "this frame", but for more precise results
   and correct ordering prefer using the pygame.MOUSEBUTTONDOWN event.
```

   The result of this function is updated when new events are processed,
   e.g. in :func:`pygame.event.get()` or :func:`pygame.event.pump()`.

   .. seealso:: :func:`pygame.mouse.get_just_released()`

   ::
      
      if pygame.mouse.get_just_pressed()[0]:
         print("LMB just pressed")

   .. versionadded:: 2.5.0

   .. ## pygame.mouse.get_just_pressed ##

.. function:: get_just_released

   | :sl:`get the most recently released buttons`
   | :sg:`get_just_released() -> (left_button, middle_button, right_button, x1_button, x2_button)`

   Similar to :func:`pygame.mouse.get_pressed()`, returing a tuple
   of length 5 with the important difference that the buttons are 

>    True only in the frame they stop to get pressed. This can be convenient

Reviewer 2: ```suggestion
   True only in the frame they stop being pressed. This can be convenient
```

I think this is clearer language for this sentence.

   for checking the buttons released "this frame" but for more precise results
   and to have the correct order the pygame.MOUSEBUTTONUP event could be

>    preferred.

Reviewer: Same here
